src/appleseed/renderer/modeling/camera/orthographiccamera.cpp

```diff
@@ -80,6 +80,15 @@ namespace
             const ParamArray&       params)
           : Camera(name, params)
         {
+            // Extract the film dimensions from the camera parameters.
+            m_film_dimensions = extract_film_dimensions();
```

Comment:
Changes in this file introduced functional regressions: it's no longer possible to change camera parameters during interactive rendering. They should be reverted.

Reply:
Ah, I believe it just needs to be in both places. This is also why the variable that @oktomus noticed was there: originally I was going to factor this code out into a dedicated function but didn't realize that the params could change between the creation and when on_render_begin is called. I'll just do that now.

Reply:
Maybe in this specific case you might be able to execute this code in the constructor, but this isn't true in general as at this point, inputs aren't bound to the entity. Why exactly do you need to do this?